inputfiles/patches/abstractworker.kdl

```diff
@@ -0,0 +1,3 @@
+interface-mixin AbstractWorker {
```

Comment:
In this specific case, I'd prefer to put all the event things to a single file as `events.kdl`, so that I can see which events are not covered by automation and can ultimately remove this file.

In this PR just the filename can change, no addition is needed.

(Because all event data technically exists, we don't have a way to track that to assign automatically right now)